koku/masu/database/report_db_accessor_base.py

```diff
@@ -112,9 +113,32 @@ def _get_db_obj_query(self, table, columns=None):
                 query = table.objects.all()
             return query
 
+    def extract_context_from_sql_params(self, sql_params: dict):
+        return {
+            "schema": sql_params.get("schema"),
+            "start_date": sql_params.get("start") or sql_params.get("start_date"),
+            "end_date": sql_params.get("end") or sql_params.get("end_date"),
```

Comment:
You could also use nested `get()` methods.
```suggestion
            "start_date": sql_params.get("start", sql_params.get("start_date")),
            "end_date": sql_params.get("end", sql_params.get("end_date")),
```

Reply:
Normally I would go with this, but I don't want a weird edge case where the `sql_params` contains: `{"start": None, "start_date": a-date, "end": None, "end_date": another-date}` because the `sql_params.get("start")` will return `None` here, regardless of what the default is.